pkg/pgmodel/extension.go

```diff
@@ -23,11 +25,42 @@ var (
 // checkExtensionVersion checks for the correct version and enables the extension if
 // it is at the right version
 func checkExtensionVersion(conn *pgxpool.Pool) error {
-	currentVersion, isInstalled, err := fetchInstalledExtensionVersion(conn)
+	timescaleVersion, isInstalled, err := fetchInstalledExtensionVersion(conn, "timescaledb")
 	if err != nil {
-		return fmt.Errorf("Could not get the installed extension version: %w", err)
+		fmt.Fprintln(os.Stderr, "could not get the installed extension version: %w", err)
+		os.Exit(1)
```

Comment:
Why are we exiting here? Can't we just return the error? (multiple places in function)

Reply:
I thought that if the Timescaledb version is in error range, then we should not allow doing operations. Fixing that to just returning errors.